lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -66,9 +66,14 @@ class SessionStateEnforcer {
       SessionMap& session_map, const std::string& imsi,
       SessionState& session_state);
 
+  /* Move to idle state */
+  void m5g_move_to_inactive_state(
+      std::string& imsi, std::unique_ptr<SessionState>& session,

```

Comment:
`const std::string&` ?

Reply:
Made the parameter as const std::string& imsi